client/driver/qemu.go

```diff
@@ -427,3 +489,23 @@ func (h *qemuHandle) run() {
 	h.waitCh <- &dstructs.WaitResult{ExitCode: ps.ExitCode, Signal: ps.Signal, Err: werr}
 	close(h.waitCh)
 }
+
+func sendQemuShutdown(logger *log.Logger, monitorPath string, userPid int) error {
+	if monitorPath == "" {
+		logger.Printf("[DEBUG] driver.qemu: monitorPath not set; will not attempt graceful shutdown for user process pid %d", userPid)
+		return errors.New("monitorPath not set")
+	} else {
+		monitorSocket, err := net.Dial("unix", monitorPath)
+		if err == nil {
```

Comment:
Swap the checking. `if err != nil { log && return err}` and de-indent the block.

Reply:
Done!